src/providers/google-vertex-ai/imageGenerate.ts

```diff
@@ -0,0 +1,77 @@
+import { GOOGLE_VERTEX_AI } from '../../globals';
+import { EmbedParams } from '../../types/embedRequestBody';
+import { ErrorResponse, ImageGenerateResponse, ProviderConfig } from '../types';
+import { generateInvalidProviderResponseError } from '../utils';
+import { GoogleErrorResponse } from './types';
+import { GoogleErrorResponseTransform } from './utils';
+
+interface GoogleImageGenParams extends EmbedParams {}
```

Comment:
Can you please verify this? ImageGenParams is extends EmbedParams seems incorrect.